lib/hint.rb

```diff
@@ -44,7 +44,10 @@ def which_machinery
     end
 
     def get_started(_options)
-      "You can get started by inspecting a system. Run:\n#{program_name} inspect HOSTNAME"
+      "You can get started by inspecting a system. Run:\n'#{program_name} inspect HOSTNAME'\n" \
+      "Specify user for connecting via SSH:\n" \
+      "'#{program_name} inspect --remote-user USER HOSTNAME'\n" \
+      "(Note: specified user needs sudo access or root on remote machine)"
```

Comment:
Maybe you can even shorten it by mention `Specify user with sudo rights for conenction via SSH:\n`